src/main/resources/transform_rules.yml

```diff
@@ -4751,8 +4747,6 @@
       data.description_values.charge_number: '[% chargeNumber %]'
       data.subcategory: alter
       original_description: '[% data.description | sentence_case %]'
-      data.action_date: '[% chargeCreationDate | bson_date %]'
-      data.description_values.charge_creation_date: '[% chargeCreationDate | bson_date %]'
```

Comment:
This doesn't look like a fall back rule, rather one which is missing an appropriate like clause. The charge_creation_date is expected for this description. https://github.com/companieshouse/api-enumerations/blob/d930c7b15828f98f1ac2f28ac704ef130fd602b1/filing_history_descriptions.yml#L463

Reply:
Restored the rule and created a new bug to cover the fix for when substitution values are in the delta's `description_values` field and not drawn from the capture groups
https://companieshouse.atlassian.net/browse/DSND-2534